From: Neil Brown <neilb@suse.de>
Date: Tue, 8 May 2007 07:12:33 +0000 (+1000)
Subject: Remove bogus add_dev definition.
X-Git-Url: http://neil.brown.name/git?p=mdadm;a=commitdiff_plain;h=8e22992203e25ae0b7bdf41cf2c53b11024d64cc

Remove bogus add_dev definition.

If nether ftw nor nftw are available, add_dev gets defined twice.
Fix that...
---

diff --git a/util.c b/util.c
index 58449d5..f57613f 100644
--- a/util.c
+++ b/util.c
@@ -468,10 +468,6 @@ int nftw(const char *path, int (*han)(const char *name, const struct stat *stb,
 	return ftw(path, add_dev_1, nopenfd);
 }
 #else
-int add_dev(const char *name, const struct stat *stb, int flag, struct FTW *s)
-{
-	return 0;
-}
 int nftw(const char *path, int (*han)(const char *name, const struct stat *stb, int flag, struct FTW *s), int nopenfd, int flags)
 {
 	return 0;
